frontend/src/metabase-types/api/mocks/settings.ts

```diff
@@ -164,6 +164,7 @@ export const createMockSettings = (
   engines: createMockEngines(),
   "has-user-setup": true,
   "hide-embed-branding?": true,
+  "hide-static-embed-terms": false,
```

Comment:
Speaking of which, what do you think about renaming this to `show-static-embed-terms` instead so the boolean flag setting name is also positive? If you look in this file, there are 2 `hide-*` flags (including this one) and `show-*` flags. It feels like we're more accustomed to using `show-*`. If we do that we'll have only 1 `hide-*` flag and 8 `show-*` flags which would look more consistent.

What do you think?

Reply:
100% agree about `show-` being easier to read